CHANGELOG.md

```diff
@@ -20,6 +61,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-NONE**
```

Comment:
This should be under the release notes for 10.1.1

Reply:
Comment addressed. Thanks